components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -1510,6 +1372,7 @@ SUBROUTINE p3_main(qc,nc,qr,nr,th_old,th,qv_old,qv,dt,qitot,qirim,nitot,birim,ss
           qidep   = qidep*icldm(i,k)  ! Vapor deposition to ice phase
           nrheti  = nrheti*rcldm(i,k) ! Change in number due to immersion freezing of rain
           nisub   = nisub*icldm(i,k)  ! Number change due to sublimation of ice
+          berg    = berg*icldm(i,k)   ! AaronDonahue, it's unclear to me what to scale berg by?
```

Comment:
berg occurs in the portion of the cloud that has ice AND liquid. This would be min(icldm,lcldm) except that we're assuming liquid and ice cloud share the same area, so it doesn't currently matter.

Reply:
excellent.  I can change this to il_cldm to be consistent.